### PR TITLE
cmd/remote-run: Support --json flag

### PR DESCRIPTION
**Pull request by adombeck**

### CODE REVIEW

File: internal/cmd/create/create.go [hunk 1]
```diff
@@ -20,11 +22,25 @@ import (
...
 	BuildSystem string `mapstructure:"build-system"`
+	Interactive bool   `mapstructure:"interactive"`
 
 	outputPath string
 	testType   config.FuzzTestType
 }
 
+func (opts *createOpts) Validate() error {
+	if opts.Interactive {
+		opts.Interactive = term.IsTerminal(int(os.Stdin.Fd())) && term.IsTerminal(int(os.Stdout.Fd()))
```
Comment (fmeum) [inline internal/cmd/create/create.go:33]: If I understand this correctly, there is jo way to force interactivity? Do you think that there could be situations where someone would need that?

File: internal/cmd/remote-run/remote_run.go [hunk 2]
```diff
@@ -132,11 +138,17 @@ https://github.com/CodeIntelligenceTesting/cifuzz/issues`, system)
 				opts.ProjectName = "projects/" + opts.ProjectName
 			}
 
+			opts.Stdout = cmd.OutOrStdout()
+			opts.Stderr = cmd.ErrOrStderr()
+
+			// If --json was specified, print all build output to stderr
+			if opts.PrintJSON {
+				opts.Stdout = cmd.ErrOrStderr()
```
Comment (fmeum) [inline internal/cmd/remote-run/remote_run.go:146]: Can we turn this into an if-else so that `opt.Stdout` is only set once in both cases?

**Review by fmeum — APPROVED** [review]

File: internal/cmd/create/create.go (hunk 1 above)
Comment (adombeck) [reply internal/cmd/create/create.go:33]: > If I understand this correctly, there is jo way to force interactivity? 

correct

> Do you think that there could be situations where someone would need that?

I can't imagine a situation where someone would need that. We're copying the behavior of `gh` here, so I think that should be fine.

File: internal/cmd/remote-run/remote_run.go (hunk 2 above)
Comment (adombeck) [reply internal/cmd/remote-run/remote_run.go:143]: done

File: internal/cmd/create/create.go (hunk 1 above)
Comment (fmeum) [reply internal/cmd/create/create.go:33]: Sounds good, just wanted to make sure we made this choice consciously.

**Review by fmeum — APPROVED** [review]

**Pull request merged**